extensions/amp-analytics/0.1/test/test-visibility-model.js

```diff
@@ -24,6 +25,13 @@ describes.sandboxed('VisibilityModel', {}, () => {
   let startTime;
   let clock;
 
+  const tick = async(timeout = 0) => {
```

Comment:
ditto: does `yield macroTask()` work here?

Reply:
It would, but I imagine that this could be used in other tests that accumulate time while avoiding testing private fields. Let me know what you think

Reply:
Actually, I think it's important that we don't yield the microtask queue when handling the `unload` event fires so I'll stick to `Promise.resolve()` and leave a comment. (I actually haven't tested it but It's my guess. We should add an E2E test for this)

Reply:
Sorry I didn't fully understand the reason why we can't yield the microTask queue here. 

The async/await and the yield with generators really look the same to me here. Could you further explain it? Thank you.